rust/scx_utils/src/topology.rs

```diff
@@ -97,6 +97,7 @@ lazy_static::lazy_static! {
 
 #[derive(Debug, Clone, Eq, Hash, PartialEq)]
 pub enum CoreType {
+    Turbo,
     Big,
```

Comment:
Would it make sense to make `Turbo` a variant of `Big`? ie. so that it's `Big { turbo: bool }`? That way, the hierarchy is encoded in the type system and people can use `match` more naturally rather than using `is_*()` helpers.

Reply:
Hm... that seems really nice, will check that!